README.md

```diff
@@ -181,9 +182,9 @@ To expedite development of [deck](https://github.com/spinnaker/deck) (or to add
 2. Run `deck` with `yarn run start` 
     * Windows users can circumvent the bash start up script by running it directly with **npm**: `npm run start-dev-server`
 
-3. Open separate terminal and SSH into your development instance:
+3. Open separate terminal and [SSH into your development instance](https://docs.aws.amazon.com/quickstarts/latest/vmlaunch/step-2-connect-to-instance.html):
 ```
-ssh -A -L 8084:localhost:8084 -L 8087:localhost:8087 ubuntu@$spinnaker_instance [-i MY_SSH_KEY]
+ssh -A -L 8084:localhost:8084 -L 8087:localhost:8087 ubuntu@$spinnaker_instance -i /path/to/my-key-pair.pem
```

Comment:
I think you need to do the same thing for the rsync command?  I just save the key in my ~/.ssh/config

Reply:
Updated